src/providers/LocalVideoProvider/docs/LocalVideoProvider.stories.mdx

```diff
@@ -14,8 +20,17 @@ The `LocalVideoProvider` provides the local video status, video tile id associat
   // Whether or not the local user has video enabled
   isVideoEnabled: boolean;
```

Comment:
May be now add one more comment to explicitly specify, this does not tell whether the video preview from `PreviewVideo` component is enabled or not and is only for the `LocalVideo` component rendered for the in-meeting screen.